.github/workflows/ci_tests.yaml

```
@@ -31,14 +31,21 @@ jobs:
           python-version: 3.8
 
       - name: Install packages
-        run: pip install black blackdoc flake8 pylint
+        run: |
+          pip install black blackdoc flake8 pylint
+          sudo apt-get install dos2unix
 
       - name: Formatting check (black and flake8)
         run: make check
 
       - name: Linting (pylint)
         run: make lint
 
+      - name: Line breaks check (dos2unix)
+        run: |
+          find . -type f -not -path '*/\.git/*' -exec grep -Iq . {} \; -exec dos2unix {} \;
```

```suggestion
          find . -type f -not -path '*/\.git/*' -exec grep -Iq . {} \; -exec dos2unix --quiet {} \;
```
Do you think we should silence all the "dos2unix: converting file" messages being printed out here in ci_tests.yaml? It's really only the diff we're interested in.